resources/migrations/000_migrations.yaml

```diff
@@ -5275,3 +5275,80 @@ databaseChangeLog:
             columns:
               - column:
                   name: parent_id
+  - changeSet:
+      id: 102
+      author: sbelak
+      changes:
+        - createTable:
+            tableName: domain_entity
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: source_table
+                  type: int
+                  constraints:
+                    nullable: false
+                    references: metabase_table(id)
+                    foreignKeyName: fk_domain_entity_ref_source_table
+                    deferrable: false
+                    initiallyDeferred: false
+              - column:
+                  name: metrics
+                  type: text
```

Comment:
do you mind adding comments/remarks for the Table and new columns that could benefit from them? It helps for people poking around the Metabase database or new people joining when they're there so they know the purpose of everything

Some of these are pretty obvious (e.g. `created_at`) so we don't really need those since they'd just be noise but for stuff like `segments` or `dimensions` you can mention that it's serialized JSON, etc.